cloud-formation-templates/ssm-document-credspec-generator.yaml

```diff
@@ -52,7 +52,7 @@ Parameters:
     Type: AWS::SSM::Parameter::Value<String>
 
   ADUserPasswordParameter:
-    AllowedPattern: (?!^([aA][wW][sS]|[sS][sS][mM]))(?=^[-a-zA-Z0-9_.]*$).*
+    AllowedPattern: (?!^([aA][wW][sS]|[sS][sS][mM]))(?=^[-a-zA-Z0-9_.\/]*$).*
```

Comment:
Almost! But parameter name should not start with either `/aws/` or `/ssm/`.
May be this might work `(?!^([/][aA][wW][sS][/]|[/][sS][sS][mM][/]))(?=^[-a-zA-Z0-9_.\/]*$).*`